cocotb_build_libs.py

```diff
@@ -294,17 +206,17 @@ def _get_vpi_lib_ext(
 def _get_vhpi_lib_ext(
     include_dir, share_lib_dir, sim_define, extra_lib=[], extra_lib_dir=[]
 ):
-    libcocotbvhpi = Extension(
-        os.path.join("cocotb", "libs", "libcocotbvhpi_" + sim_define.lower()),
+    libcocotbvhpi = DSO(
+        dso_join("cocotb", "libs", "cocotbvhpi_" + sim_define.lower()),
```

Comment:
Presumably these no longer end up in subdirectories? If not, where are they installed to?

Reply:
They do end up in the same path. setuptools_dso converts from `cocotb.libs.name` to `cocotb/libs/libname.so`, `cocotb/libs/name.dll`, `cocotb/libs/libname.dylib`

Reply:
Mind adding a comment to that effect in `dso_join`?